package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
It works when I build post-robot using src/ and just plain cross-domain-utils.  For minification I don't think you would export that because the parent module would do minification.